data/json/recipes/recipe_food.json

```diff
@@ -6345,7 +6381,20 @@
     "autolearn": true,
     "batch_time_factors": [ 83, 3 ],
     "tools": [ [ [ "food_processor", 20 ] ] ],
-    "components": [ [ [ "corn", 1 ], [ "irradiated_corn", 1 ], [ "kernels", 1 ] ] ]
+    "components": [ [ [ "dry_corn", 1 ] ] ]
+  },
+  {
+    "result": "corn_kernels",
+    "type": "recipe",
```

Comment:
(I don't know) can this be a Disassembly action?

Reply:
I'd rather it not be, so that people will know you can do it... but I can be convinced, I'm ambivalent either way

Reply:
Would it be detrimental in any way if it was both? Imagine you find a single corn ear in the game. With disassembly you can just get the kernels immediately and throw away the cob (can it be autodropped like containers?). With recipe you have to open the recipe interface, search for the needed recipe among thousands of others, maybe drop what you are holding to take the temporary `craft` item into your hands...

Reply:
To me it makes more sense that you would benefit from having a place to work on when you are peeling corn, it's not something you can do anywhere.

Reply:
Yes, that's another valid scenario.